src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -158,15 +159,30 @@ final case class PurposeApiServiceImpl(
     val operationLabel = s"Retrieving Purpose $id"
     logger.info(operationLabel)
 
+    def tryToValidateRiskAnalysisForm(
+      riskAnalysisForm: Option[PurposeManagementDependency.RiskAnalysisForm]
+    ): Future[Option[Boolean]] = {
+      Future.successful(
+        riskAnalysisForm
+          .map(RiskAnalysisConverter.dependencyToApi(_))
+          .map(
+            RiskAnalysisValidation
+              .validate(_, schemaOnlyValidation = false)
+              .isValid
+          )
+      )
+    }
+
     val result: Future[Purpose] = for {
-      organizationId <- getOrganizationIdFutureUUID(contexts)
-      uuid           <- id.toFutureUUID
-      purpose        <- purposeManagementService.getPurpose(uuid)
-      eService       <- catalogManagementService.getEServiceById(purpose.eserviceId)
+      organizationId          <- getOrganizationIdFutureUUID(contexts)
+      uuid                    <- id.toFutureUUID
+      purpose                 <- purposeManagementService.getPurpose(uuid)
+      isRiskAnalysisFormValid <- tryToValidateRiskAnalysisForm(purpose.riskAnalysisForm)
+      eService                <- catalogManagementService.getEServiceById(purpose.eserviceId)
       authorizedPurpose =
         if (organizationId == purpose.consumerId || organizationId == eService.producerId) purpose
         else purpose.copy(riskAnalysisForm = None) // Hide risk analysis to other organizations
```

Comment:
we should always return `false` here, where the case an organization is not allowed to access the risk analysis